doc/guide/guide-dynamics.rst

```diff
@@ -9,6 +9,7 @@ Time Evolution and Quantum System Dynamics
 
    dynamics/dynamics-intro.rst
    dynamics/dynamics-data.rst
+   dynamics/dynamics-dysolve-propagator.rst
```

Comment:
Maybe move it after `dynamics/dynamics-propagator.rst`?
But it does not belong before `dynamics/dynamics-master.rst`.